<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tab_txt</class>
 <widget class="QWidget" name="tab_txt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_decripted">
     <property name="text">
      <string>原文：</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QPlainTextEdit" name="textedit_decripted"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_key">
     <property name="text">
      <string>密钥：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineedit_key">
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QCheckBox" name="ckb_vertify">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>原文校验</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_encripted">
     <property name="text">
      <string>密文：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QPlainTextEdit" name="textedit_encripted"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
